### PR TITLE
Add `Authorization and authentication` in `info.description`

### LINKED ISSUE #212: Add "Authorization and authentication" - section in `info.description`
**Problem description**
We need to add the ["Authorization and authentication" - section in "info.description"](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#use-of-openidconnect-for-securityschemes) in all specifications.

### PR DESCRIPTION
**Pull request by maxl2287**
#### What type of PR is this?

Add one of the following kinds:
* documentation

#### What this PR does / why we need it:
Adds the `Authorization and authentication` to the `info.description`

#### Which issue(s) this PR fixes:

Fixes #212

### CODE REVIEW

**Review by bigludo7 — DISMISSED** [review]
LGTM
Thanks

File: code/API_definitions/geofencing-subscriptions.yaml [hunk 1]
```diff
@@ -57,6 +57,14 @@ info:
       Developers may provide a callback URL on which notifications regarding geofencing can be received from the service provider.
       If an event occurs the application will send events to the provided webhook - 'notificationUrl'._
 
+    ### Authorization and authentication
```
Comment (fernandopradocabrillo) [inline code/API_definitions/geofencing-subscriptions.yaml:60]:
```suggestion
    # Authorization and authentication
```

File: code/API_definitions/location-retrieval.yaml [hunk 2]
```diff
@@ -52,6 +52,14 @@ info:
         * a localization defined with a circle with center specified by the latitude and longitude, and radius for answer accuracy,
         * a timestamp about location information freshness.
 
+    ### Authorization and authentication
```
Comment (fernandopradocabrillo) [inline code/API_definitions/location-retrieval.yaml:55]:
```suggestion
    # Authorization and authentication

```

File: code/API_definitions/location-verification.yaml [hunk 3]
```diff
@@ -46,6 +46,14 @@ info:
 
     - Verify whether the device location is within a requested area, currently a circle with center specified by the latitude and longitude, and radius specified by the accuracy. The operation returns a verification result and, optionally, a match rate estimation for the location verification in percent.
 
+    ### Authorization and authentication
```
Comment (fernandopradocabrillo) [inline code/API_definitions/location-verification.yaml:49]:
```suggestion
    # Authorization and authentication
```

**Review by fernandopradocabrillo — COMMENTED** [review]
These are just small cosmetic comments since the section hierarchy is not consistent across APIs.

File: code/API_definitions/geofencing-subscriptions.yaml (hunk 1 above)
Comment (maxl2287) [reply code/API_definitions/geofencing-subscriptions.yaml:60]: I am okay with that, but should we update this as well in Commonalities? 

https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#mandatory-template-for-infodescription-in-camara-api-specs

Comment (bigludo7) [reply code/API_definitions/geofencing-subscriptions.yaml:60]: @fernandopradocabrillo Probable good to raise an issue to ICM team to propose the correction - wdyt?

**Review by bigludo7 — APPROVED** [review]
LGTM

**Comment by maxl2287** [comment]
@bigludo7 shell I create a ticket on Commonalities for the section hierarchy of this chapter?

**Comment by bigludo7** [comment]
> @bigludo7 shell I create a ticket on Commonalities for the section hierarchy of this chapter?

Hi @maxl2287 - Yes probably to fix this once. Thanks !

**Review by fernandopradocabrillo — APPROVED** [review]
LGTM

thanks @maxl2287 !

**Pull request merged**